docs/content/en/docs/getting-started/production-environment/snow-getstarted.md

```diff
@@ -0,0 +1,227 @@
+---
+title: "Create Snow production cluster" 
+linkTitle: "Snow cluster" 
+weight: 40
+description: >
+  Create a production-quality cluster on AWS Snow
+---
+
+EKS Anywhere supports an AWS Snow provider for production grade EKS Anywhere deployments.
+
+This document walks you through setting up EKS Anywhere on Snow as a standalone, self-managed cluster or combined set of management/workload clusters.
+See [Cluster topologies]({{< relref "../../concepts/cluster-topologies" >}}) for details.
+
+## Prerequisite checklist
+
+EKS Anywhere on Snow needs:
+
+* EKS Anywhere enabled Snowball devices. See [Ordering EKS anywhere enabled Snow Devices](https://docs.aws.amazon.com/snowball/latest/ug/whatissnowball.html) for ordering experience through the AWS Snow Family console.
+* To be run on an Admin instance in a Snowball Edge device. See [Initial Device Setup](https://docs.aws.amazon.com/snowball/latest/ug/whatissnowball.html) for setting up the devices, launching the Admin instance, fetching and copying the device credentials to the admin instance for `eksctl` CLI to consume.
+
+Also, see the [Ports and protocols]({{< relref "/docs/reference/ports.md" >}}) page for information on ports that need to be accessible from control plane, worker, and Admin machines.
```

Comment:
Should we do a PR to add a list of ports that need to be opened on the Snow device, to the Ports and Protocols doc, as we do with other providers?

Reply:
added snow provider in ports and protocols section